web_view_searchpanel/static/src/js/kanban_controller.js

```diff
@@ -0,0 +1,58 @@
+/**
+*
+*    Copyright 2017-2019 MuK IT GmbH.
+*    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+*
+**/
+
+odoo.define('web_biew_searchpanel.KanbanController', function (require) {
```

Comment:
and the same for the rest

```suggestion
odoo.define('web_view_searchpanel.KanbanController', function (require) {
```

Reply:
Sorry, I had the search with a defined kind of files and I missed them all :disappointed: